mpp/include/mpp_util.inc

```diff
@@ -124,6 +124,8 @@
 !$OMP end single
 !$  endif
 
+    ! Set to negative value for default/bad unit
+    stdlog = -1
```

Comment:
To me, this is a bug in gfortran.  In every case, stdlog will be defined on exit from the routine and there is no attempt to use stdlog prior to it being set.